### ISSUE TITLE
[Dracula Pro] Indent guide is hard to see

### ISSUE THREAD

**Issue by ayuthmang** [dracula/dracula-theme#980]
**Describe the bug**
The indent guide in VS Code is hard to distinguish.

<img width="710" alt="image" src="https://github.com/user-attachments/assets/664b9c3d-4dcf-41d5-ad36-d1e7bd3cc52a">

- without highlighting, we are unable to see the indent guideline compared to Dracula Open Source is more distinguished and clear to see the indent guide.

**To Reproduce**
1. Apply the Dracula Pro theme
2. Open File Explorer and see a subfolder

**Expected behavior**
The indent guide should be distinguished.

**Screenshots**

**Platform (please complete the following information):**
 - Theme: VS Code
 - Theme Filter: Pro, Alucard
 - Theme version: 2.2
 - OS: macOS
 - Version 14.6.1

**Comment by praxxis** [comment]
Agreed, the indentation lines in Pro (at least for Van Helsing) are pretty much invisible. There's a guide [here](https://www.roboleary.net/2021/11/06/vscode-you-dont-need-that-extension2.html#3-indentation-guides-colorization) on how to customize the colors for the editor. For the file tree you can alter `tree.indentGuidesStroke` and `tree.inactiveIndentGuidesStroke`.

**Comment by luxonauta** [comment]
Fixed for next release. Thanks for the report! 🧩
